client/scripts/views/pages/search.ts

```diff
@@ -231,43 +240,60 @@ const SearchPage : m.Component<{
       });
     }
 
-    if (!app.searchCache[ALL_RESULTS_KEY]?.loaded) {
-      return LoadingPage;
-    }
-
     // re-fetch results for new search if search term or URI has changed
-    if (searchTerm !== vnode.state.searchTerm || vnode.state.refreshResults) {
-      vnode.state.searchTerm = searchTerm;
+    if (!_.isEqual(searchQuery, vnode.state.searchQuery) || vnode.state.refreshResults) {
+      vnode.state.searchQuery = searchQuery;
       vnode.state.refreshResults = false;
       vnode.state.results = {};
-      if (!app.searchCache[vnode.state.searchTerm]) {
-        app.searchCache[vnode.state.searchTerm] = { loaded: false };
-      }
-      search(searchTerm, { communityScope, chainScope }, vnode.state);
+      search(searchQuery, vnode.state);
       return LoadingPage;
     }
 
-    if (!app.searchCache[searchTerm].loaded) {
+    if (!app.search.getByQuery(searchQuery).loaded) {
```

Comment:
Getting an error thrown here; `.loaded` is getting called on a null value—try `?.` (not sure if that'll cover the functionality tho)

Reply:
This was also caused by the url bug as far as I can tell. Let me know if it still shows up for you